src/commands/CmdImport.h

```diff
@@ -31,6 +31,7 @@
 #include <JSON.h>
 
 #include <string>
+#include <unordered_map>
```

Comment:
Another include-what-you-use issue, I think?

Reply:
Most likely yes. :D